dockstore-webservice/src/main/java/io/dockstore/webservice/core/SourceFile.java

```diff
@@ -139,12 +147,30 @@ public void setType(DescriptorLanguage.FileType type) {
         this.type = type;
     }
 
+    @ApiModelProperty(value = "Cache for the contents of the target file", position = 2)
```

Comment:
I think at the time this was written, we weren't totally decided on keeping a copy of the file. 
We used the term cache to indicate that this is our copy of it and that it might be out of sync with the "gold" (target) copy of the file on GitHub